README.adoc

```diff
@@ -1566,13 +1562,14 @@ try(final SingleChronicleQueue q1 = SingleChronicleQueueBuilder.binary("queue-st
 }
 ----
 
-The method shutdown(), closes the queue and releases any other resources.
+The method `close()`, closes the Pretoucher and releases its resources.
```

Comment:
Why talk about it in CQ? Maybe just mention we have Pretoucher in Enterprise.

Reply:
Yes fair point